lib/gollum-lib/macro/video.rb

```diff
@@ -0,0 +1,17 @@
+module Gollum
+  class Macro
+    class Video < Gollum::Macro
+      def render (fname)
+        begin 
```

Comment:
Do we need this elaborate way of determining whether the video file exists? If the video tag is passed a path to a non-existent file, what will happen? Won't the browser just handle it as a broken link?